src/app/card-list/card-list.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit } from '@angular/core';
+import { CardComponent } from '../card';
+
+
+@Component({
+  moduleId: module.id,
+  selector: 'app-card-list',
+  templateUrl: 'card-list.component.html',
+  styleUrls: ['card-list.component.css'],
+  directives:[CardComponent]
```

Comment:
remove directives and move 'CardComponent' to the module file in the 'declarations' area